### ISSUE TITLE
Backward pass fail when training se_resnext50_32x4d

### ISSUE THREAD

**Issue by wmmxk** [CeLuigi/models-comparison.pytorch#2]
When training the model, I run into this error: one of the variables needed for gradient computation has been modified by an inplace operation. I kind of understand what the error means after googling, but there are so many variables in the definition of the class. So do you know any potential culprit? Thanks.

**Comment by CeLuigi** [comment]
The released code has not been tested for training the models, because we compared pre-trained models. However, I have just run the [following training code](https://github.com/kuangliu/pytorch-cifar) on CIFAR-100 database using the [se_resnext50_32x4d](https://github.com/CeLuigi/models-comparison.pytorch/blob/master/pretrainedmodels/models/senet.py) model without any error. Unfortunately I do not know the source of your issue.
